components/Tabs/Tabs.js

```diff
@@ -1,38 +1,42 @@
-
 class TabItem {
   constructor(element) {
-    // attach dom element to object. Example in Tabs class
+    this.element = element;// attach dom element to object. Example in Tabs class
```

Comment:
I would suggest to add a space between the semicolon and the start of your comment.